src/pandas_profiling/profile_report.py

```diff
@@ -136,6 +139,77 @@ def __initialize_dataframe(
         else:
             return df
 
+    def _timeseries_heatmap(
```

Comment:
@alexbarros the heatmap function should be an auxiliary function which means shoudn't belog to the ProfileReport class. 

Please add it as a separate function to another python file to be made available independently from the ProfileReport. Let's keep the class method to be related to the Report itself like to_file, etc.